src/js/blocks/hero/block.js

```diff
@@ -0,0 +1,379 @@
+const { __ } = wp.i18n;
+const { registerBlockType } = wp.blocks;
+const {
+	Dashicon,
+	IconButton,
+	PanelBody,
+	PanelRow,
+	Toolbar,
+	withNotices,
+	BaseControl,
+	TextControl,
+	Button,
+	ButtonGroup,
+	ToggleControl
+} = wp.components;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['wp' is not defined. (no-undef)](https://app.codacy.com/manual/uriweb/uri-component-library/pullRequest?prid=4757242)